app/assets/stylesheets/application.css.scss

```diff
@@ -53,6 +53,16 @@ body {
   color: #fff;
 }
 
+.link-to-text-color:hover {
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.